src/app/components/SubjectHeading/Search/SubjectHeadingSearch.jsx

```diff
@@ -19,6 +19,20 @@ class SubjectHeadingSearch extends React.Component {
     this.onSubmit = this.onSubmit.bind(this);
     this.resetAutosuggest = this.resetAutosuggest.bind(this);
     this.changeActiveSuggestion = this.changeActiveSuggestion.bind(this);
+    this.onFocus = this.onFocus.bind(this);
+  }
+
+  componentDidMount() {
```

Comment:
it's too bad this can't be done using `onBlur`

Reply:
Yeah, I would have preferred that but I can't figure out how to make it work.